src/discussions/posts/post/PostLink.test.jsx

```diff
@@ -15,7 +15,7 @@ function renderComponent(post) {
   return render(
     <IntlProvider locale="en">
       <AppProvider store={store}>
-        <PostLink post={post} key={post.id} isSelected={() => true} />
+        <PostLink post={post} key={post.id} isSelected={() => true} learnerTab />
```

Comment:
all test cases here assuming component called from learner tab? whereas this prop is not required to be passed in the component. why do we need that change in test cases?

Reply:
Yes, you are right. I'll make it dynamic which will cover both cases leaner and simple posts.